Libraries/Hotcakes.Commerce/Common/Constants.cs

```diff
@@ -163,5 +163,42 @@ public class Constants
         public const string TAG_IOGPRICEAMOUNT = "<meta property=\"og:price:amount\" content=\"{0}\" />";
         public const string TAG_IOGPRICECURRENCY = "<meta property=\"og:price:currency\" content=\"{0}\" />";
         #endregion
+
+        #region PayPal
```

Comment:
This is looking good, @jldelatorre ...  If you choose to put the constants in this global constants class, we'd want each constant to be preceeded with `PAYPAYEXP_`.  However, each of these constants are only relevant to and for PayPal.  

So, this either needs to be in a new PayPal-only constants class, or in the same class file.  Either is fine, but should be dictated on the scope of the constants.  

For example, if the constants might be used for both Express and Pro, then they should probably be in a single class file, accessed by both PayPal Express & Pro payment options.  If the constants are different or not used in both, then maybe they go into the same class file as the rest of the code.